cpp-client/build.gradle

```diff
@@ -34,23 +35,6 @@ tasks.register('updateProtobuf', Sync) {
     into protoSourceDir
 }
 
-license {
-    header rootProject.file('license-header')
-    include '**/*.h'
-    include '**/*.cc'
-    strictCheck true
-    useDefaultMappings true
-    ignoreFailures true
-}
-tasks.getByName('license').dependsOn(tasks.create("licensePy", com.hierynomus.gradle.license.tasks.LicenseCheck) {
-    source = ['deephaven']
-})
-tasks.getByName('licenseFormat').dependsOn(tasks.create("licenseFormatPy", com.hierynomus.gradle.license.tasks.LicenseFormat) {
-    source = ['deephaven']
-})
-
-project.tasks.getByName('quick').dependsOn project.tasks.getByName('license')
```

Comment:
More license stuff is dropped.

Reply:
Yep, by design, py/cpp/proto/etc will be managed by spotless once we re-enable that.